test/index.js

```diff
@@ -698,6 +698,212 @@ describe("mongoose_delete with options: { deletedId : true, deletedIdType: Strin
     });
 });
 
+describe("mongoose_delete with session management", function () {
+  var TestSchema = new Schema({ name: String }, { collection: 'mongoose_delete_test_sessions' });
+  TestSchema.plugin(mongoose_delete, { deletedAt: true, deletedBy: true, deletedId: true, overrideMethods:'all' });
+  var TestModel = mongoose.model('TestSession', TestSchema);
+
+  beforeEach(async function () {
+    await TestModel.create([
+      { name: 'Anakin Skywalker' },
+      { name: 'Obi-Wan Kenobi' },
+      { name: 'Yoda' }
+    ]);
+  });
+
+  afterEach(async function () {
+    await mongoose.connection.db.dropCollection("mongoose_delete_test_sessions");
+  });
+
+  const userId = getNewObjectId("53da93b16b4a6670076b16bf");
+  const actionId = getNewObjectId("53da93b16b4a6670076b16c0");
+
+  it("delete() -> should support transactions with session parameter", async function () {
```

Comment:
🤌 The title of the test doesn't describe the behaviour here
```suggestion
  it("delete() -> should support transaction rollbacks with session parameter", async function () {
```

Reply:
❓ Does the test suite allow us to do assertion counting checks, e.g.
```suggestion
  it("delete() -> should support transactions with session parameter", async function () {
    expect.assertions(5)
```